### PR TITLE
Release Version 1.2.0

### PR DESCRIPTION
**Pull request by oxzi**
Part of #707. Should be merged last before drafting the release.

### CODE REVIEW

File: CHANGELOG.md
Comment (julianbrost) [inline CHANGELOG.md:15]: It's done like last time, but that PR list is very long and doesn't look helpful at all. Maybe just link to https://github.com/Icinga/icingadb/pulls?q=is%3Apr+milestone%3A1.1.2+label%3Adependencies instead?

File: CHANGELOG.md
Comment (julianbrost) [inline CHANGELOG.md:1]: I'd stay there's still some improvement possible regarding understandability for end users, i.e. saying that `wsrep_sync_wait` is for Galera clusters, `Flatten()` for custom variables, etc.

Comment (oxzi) [reply CHANGELOG.md:1]: I added initial prose for essential changes and reworded some of the bullet points to make the change's impact more obvious.

**Review by julianbrost — COMMENTED** [review]
Should be fine, I'm waiting for everything else to be done for a final approval though.

File: CHANGELOG.md
Comment (julianbrost) [inline CHANGELOG.md:22]: Maybe this should also get a funny little R, see #736.

Comment (oxzi) [reply CHANGELOG.md]: Added the funny little ᴙ and linked to the trademark document.

<sub><sup>~~As their trademark policy is not RFC 2119 compliant, section 5.a i. could be ignored.~~</sup></sub>

**Review by julianbrost — CHANGES_REQUESTED** [review]
The PR has to be adapted to the short notice PRs. At least #739 should be mentioned as `EPIPE` is mentioned explicitly already. (There might be more noteworthy ones, I haven't checked all so far.)

**Comment by oxzi** [comment]
@julianbrost: I rebased this PR's branch onto main. Afterwards, I reworded the CHANGELOG to group similar changes (especially everything retry-related) and include the latest additions.

**Review by yhabteab — APPROVED** [review]

**Comment by julianbrost** [comment]
> Afterwards, I reworded the CHANGELOG to group similar changes (especially everything retry-related) and include the latest additions.

Looks like you have lost #692 in the process.

**Comment by oxzi** [comment]
> Looks like you have lost #692 in the process.

This was intentional, as, on second thought, I found this change to be irrelevant for a CHANGELOG entry.

**Review by julianbrost — APPROVED** [review]

**Review by lippserd — APPROVED** [review]

**Pull request merged**
